imu_topic: /livox/imu
lidar_topic: /livox/lidar
body_frame: imubody
world_frame: local
print_time_cost: true
ros_spin_thread: 1
# lidar_type: livox、robosense
lidar_type: livox

lidar_filter_num: 1
lidar_min_range: 0.5
lidar_max_range: 30.0
scan_resolution: 0.2
map_resolution: 0.2

cube_len: 300
det_range: 60
move_thresh: 1.5

na: 0.01
ng: 0.01
nba: 0.0001
nbg: 0.0001

imu_init_num: 20
near_search_num: 5
ieskf_max_iter: 5

gravity_align: true
esti_il: false

r_il: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
t_il: [-0.011, -0.02329, 0.04412]

lidar_cov_inv: 1000.0

# 车体系
carbody_frame: carbody
# 激光本体系
lidarbody_frame: lidarbody
t_carbody_lidar: [ 0.35, 0., 0. ]
r_carbody_lidar: [-1.00000, -0.00000, 0.00000, 
                  0.00000, -1.00000, 0.00000, 
                  0.00000, 0.00000, 1.00000]
